src/Microsoft.Identity.Abstractions/DownstreamApi/IDownstreamApi.HttpMethods.cs

```diff
@@ -44,6 +45,9 @@ public partial interface IDownstreamApi
         ///         });
         /// </code>
         /// </example>
+#if NET6_0_OR_GREATER
+        [RequiresUnreferencedCode("Provide source generation for TOutput")]
```

Comment:
```suggestion
        [RequiresUnreferencedCode("Provide source generation for TOutput. See https://aka.ms/ms-id-web/il-trimming/generator")]
```

Reply:
I don't see it resolved?

Reply:
I changed the template and thought it had run. The changes should be there now.